### ISSUE TITLE
Support additional ELB properties in ElasticLoadBalancer component

### ISSUE THREAD

**Issue by hjacobs** [zalando-stups/senza#105]
It should be possible to specify ELB IdleTimeout and other properties, see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-elb.html

**Comment by hjacobs** [comment]
This was fixed in version 0.74 (https://github.com/zalando-stups/senza/releases/tag/0.74)

You can now pass additional properties in the Senza definition, e.g. to set the idle timeout to 300s:
```
  - AppLoadBalancer:
      Type: Senza::WeightedDnsElasticLoadBalancer
      HTTPPort: 8080
      HealthCheckPath: /health
      SecurityGroups:
        - app-connection-tracker-lb
      ConnectionSettings:
        IdleTimeout: 300

**Comment by hjacobs** [comment]
hi, i can't create new stacks when i use 'MainDomain' and/or 'VersionDomain' under "AppLoadBalancer" anymore - i just updated the stups tools. But they are still in the readthedocs documentation. are they renamed?

 uh, that's a bug I introduced

**Comment by hjacobs** [comment]
Fixed in version 0.76 https://github.com/zalando-stups/senza/releases/tag/0.76
